tests/controller/insertcontent.js

```diff
@@ -610,6 +612,18 @@ describe( 'DataController', () => {
 				insertHelper( '<disallowedWidget>xxx</disallowedWidget>' );
 				expect( getData( doc ) ).to.equal( '<paragraph>f[]oo</paragraph>' );
 			} );
+
+			it( 'filters out disallowed attributes', () => {
```

Comment:
Just a single test? I can't believe that this ticket was so simple ;)

Reply:
Hm... I can't think of anything else right now. I was worried that some falsy cases might not be covered, but I see that they are. So, it LGTM.